app/src/main/java/org/sdc/ttu/sdc_test_game_app/Login/LoginActivity.java

```diff
@@ -88,6 +90,8 @@ public void onLoginClick(View view) {
 
     @OnClick(R.id.text_create_account)
     public void onCreateAccountClick(View view) {
+        Intent createAccount = new Intent(this, CreateAccountActivity.class);
+        mPresenter.createActivity(createAccount);
```

Comment:
Change the method in `mPresenter` to take in `Context` as an argument instead of an Intent, so we can keep the view as dumb as possible.